src/bank.rs

```diff
@@ -649,6 +649,15 @@ impl Bank {
                 self.par_execute_entries(&mt_group)?;
                 mt_group = vec![];
                 //reset the lock and push the entry
+                locked
```

Comment:
I think you can just call unlock_accounts() directly...  unlock_account() checks the result, and unlocks iff the failure wasn't "failure to lock", i.e AccountInUse

Reply:
👍